ompi/include/mpi.h.in

```diff
@@ -1389,7 +1389,7 @@ OMPI_DECLSPEC extern struct ompi_predefined_datatype_t ompi_mpi_ub;
 /*
  * MPI API
  */
-
+#ifndef OMPI_NO_MPI_PROTOTYPES
```

Comment:
This is left-over from the standard ABI code; I can remove it for now.

IIRC the reason I didn't use a 0/1 macro there is because it would require adding the macro in mpicc or forcing the end user to add it. Both the standard ABI and the ompi ABI code were using mpi.h since some types defined there needed to be used in both versions, but the prototypes cannot be the same.

When we start working again on the standard ABI code, it might be better to refactor mpi.h out into multiple files to make this easier.

Reply:
Ok, gotcha -- those are good reasons.  If you end up keeping something like this, having a comment to explain the deviation from our norm (i.e., using `#ifndef`) would be good.